content/influxdb/v2.0/query-data/common-queries/recalculate_value_column.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Recalculate the `_value` column in place
+seotitle: Recalculate the `_value` column in place
+description: Recalculate the `_value` column without creating a new one.
+influxdb/v2.0/tags: [queries]
+menu:
+  influxdb_2_0:
+    name: Recalculate the `_value` column
+    parent: Common queries
+weight: 104
+---
```

Comment:
~~~yaml
---
title: Recalculate the _value column
description: Recalculate the `_value` column without creating a new one.
influxdb/v2.0/tags: [queries]
menu:
  influxdb_2_0:
    name: Recalculate the _value column
    parent: Common queries
weight: 104
---
~~~